### PR TITLE
added chat sound on receiving a message

### PR DESCRIPTION
**Pull request by poterstar**

### CODE REVIEW

File: ui/chat/chat.svelte [hunk 1]
```diff
@@ -15,6 +16,9 @@
   grooming.onChange(
     state => {
       messages = state.chat;
+      if (state.chat[state.chat.length-1].author !== grooming.username) {
```
Comment (gaswelder) [inline ui/chat/chat.svelte]: This can crash if state.chat.length is 0.

File: ui/chat/chat.svelte [hunk 2]
```diff
@@ -15,6 +16,9 @@
...
+        audio.play();
```
Comment (gaswelder) [inline ui/chat/chat.svelte:30]: What if that audio URL below becomes invalid? Will `audio.play` throw an error?
What if `audio.play` is called before the file is loaded?

Comment (poterstar) [reply ui/chat/chat.svelte:30]: Sound will not play and we can see an error in the console, that is all, the app will not be crashed. I added check that audio is available for playing.

File: ui/chat/chat.svelte (hunk 1 above)
Comment (poterstar) [reply ui/chat/chat.svelte]: fixed

**Pull request merged**